packages/ui/src/overview/hooks/useDeadlines.ts

```diff
@@ -33,13 +41,15 @@ export const useDeadlines = (params?: UseDeadlinesParams) => {
 
   const deadlines = useMemo((): Record<DeadlineNamespace, any[]> => {
     const rawElection = data?.electionRounds[0]
-    const election = rawElection && asElection(rawElection)
+    const election = rawElection && [asElection(rawElection)]
     const proposals = data?.proposals || []
-    const upcomingOpenings = data?.upcomingWorkingGroupOpenings.map(asUpcomingWorkingGroupOpening) || []
+    const upcomingOpenings = data?.upcomingWorkingGroupOpenings?.map(asUpcomingWorkingGroupOpening) || []
     const openings = data?.workingGroupOpenings || []
     return {
       proposals: proposals.filter((proposal) => !storageDeadlines.proposals.includes(proposal.id)),
-      election: election?.candidates.filter((candidate) => !storageDeadlines.election.includes(candidate.id)) ?? [],
+      elections:
+        election?.filter((election) => !storageDeadlines.elections.includes(`${election.cycleId}:${electionStage}`)) ??
```

Comment:
Because `electionStage` is used here, it would be safer to add it to the dependencies of this `useMemo`